File: tests/fixtures/vulnerable/cmdi.py
# FIXTURE: OS command injection, Python.
import os
import subprocess

def ping(host):
    # EXPECT command-injection
    os.system("ping -c 1 " + host)

def list_dir(directory):
    # EXPECT command-injection
    subprocess.run("ls -la " + directory, shell=True)

File: tests/fixtures/vulnerable/cmdi.js
// FIXTURE: OS command injection, JavaScript.
const { exec, spawn } = require('child_process');

function pingHost(host) {
  // EXPECT command-injection
  exec("ping -c 1 " + host, (err, stdout) => console.log(stdout));
}

function listDir(dir) {
  // EXPECT command-injection
  spawn("ls -la " + dir, { shell: true });
}
